tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Workbenches.resource

```diff
@@ -0,0 +1,244 @@
+    Wait Until Page Contains Element    xpath=//ul[@role="listbox"]/li/button[text()="${k8s_type}"]
+    Click Element    xpath=//ul[@role="listbox"]/li/button[text()="${k8s_type}"]
+    Run Keyword And Continue On Failure    Wait Until Page Contains Element    xpath=//section[@aria-label="Environment variables"]//button[div/span[text()="Select one"]]
+    Click Element    xpath=//section[@aria-label="Environment variables"]//button[div/span[text()="Select one"]]
+    Wait Until Page Contains Element    xpath=//ul[@role="listbox"]/li/button[text()="${input_type}"]
+    Click Element    xpath=//ul[@role="listbox"]/li/button[text()="${input_type}"]
+
+Remove All Environment Variables
+    ${remove_buttons}=    Get WebElements    xpath=//section[@id="environment-variable"]//button
+    FOR    ${rb}    IN    @{remove_buttons}
+        Click Element    ${rb}
+    END
+
+Select An Existent PV
+    [Arguments]    ${name}
+    Run Keyword And Continue On Failure    Wait Until Element Is Enabled    xpath=//input[@placeholder="Select a persistent storage"]
+    Click Element    xpath=//input[@placeholder="Select a persistent storage"]
+    Wait Until Page Contains Element    xpath=//ul/li/button[text()="${name}"]
+    Wait Until Page Contains Element    xpath=//div[contains(@class,"pf-c-select")]/ul/li/button[text()="${name}"]
+    Click Element   xpath=//ul/li/button[text()="${name}"]
+
+Select Workbench Jupyter Image
+    [Arguments]     ${image_name}
+    Wait Until Page Contains Element    ${WORKBENCH_IMAGE_MENU_BTN_XP}
+    Click Button    ${WORKBENCH_IMAGE_MENU_BTN_XP}
+    Wait Until Page Contains Element    ${WORKBENCH_IMAGE_ITEM_BTN_XP}/span[text()="${image_name}"]
+    Click Element    ${WORKBENCH_IMAGE_ITEM_BTN_XP}/span[text()="${image_name}"]
+
+Select Workbench Container Size
+    [Arguments]     ${size_name}=Small
+    Wait Until Page Contains Element    ${WORKBENCH_SIZE_MENU_BTN_XP}
+    Click Button    ${WORKBENCH_SIZE_MENU_BTN_XP}
+    Wait Until Page Contains Element    ${WORKBENCH_SIZE_ITEM_BTN_XP}/span[text()="${size_name}"]
+    Click Element    ${WORKBENCH_SIZE_ITEM_BTN_XP}/span[text()="${size_name}"]
+
+Workbench Should Be Listed
+    [Arguments]     ${workbench_title}
+    Run keyword And Continue On Failure     Wait Until Page Contains Element    ${WORKBENCH_SECTION_XP}//td[@data-label="Name"]/h4[text()="${workbench_title}"]
+
+Workbench Should Not Be Listed
+    [Arguments]     ${workbench_title}
+    Run keyword And Continue On Failure     Wait Until Page Does Not Contain Element    ${WORKBENCH_SECTION_XP}//td[@data-label="Name"]/h4[text()="${workbench_title}"]
+
+Workbench Status Should Be
+    [Arguments]     ${workbench_title}      ${status}
+    Run keyword And Continue On Failure     Page Should Contain Element    ${WORKBENCH_SECTION_XP}//tr[td[@data-label="Name"]/h4[text()="${workbench_title}"]]/td[@data-label="Status"]/label/span[text()="${status}"]
```

Comment:
```suggestion
    Run keyword And Continue On Failure     Page Should Contain Element    ${WORKBENCH_SECTION_XP}//tr[td[@data-label="Name"]/h4[div[text()="${workbench_title}"]]]/td[@data-label="Status"]//p[text()="${status}"]
```

there are multiple like that, I had to replace `h4[text()="${workbench_title}"]` by `h4[div[text()="${workbench_title}"]]`

Reply:
ah interesting, that must be some late changes in html..hm okay, thank you I will fix it!

Reply:
yes, clearly some details in the HTML changed since you wrote the code! I comment on everything I have to change, that will save you a few minutes when you get back to it ;-)

Reply:
absolutely, you spared me time! I did the changes and it worked. Thx!